### PR TITLE
[Merged by Bors] - feat: `Record` instance getters for timestamps

### PR DESCRIPTION
**Pull request by EstebanBorai**
Getters for `self.header.first_timestamp` and ` self.preamble.offset_delta`

---

Part of: https://github.com/infinyon/fluvio/pull/3359

### CODE REVIEW

**Review by morenol — APPROVED** [review]
LGTM

File: crates/fluvio-protocol/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [package]
 name = "fluvio-protocol"
 edition = "2021"
-version = "0.10.3"
+version = "0.11.0"
```
Comment (sehz) [inline crates/fluvio-protocol/Cargo.toml:4]: the changes doesn't seem like need to bump major version

Comment (EstebanBorai) [reply crates/fluvio-protocol/Cargo.toml:4]: Will do minor!

**Review by sehz — APPROVED** [review]

**Pull request closed without merging**